pkg/scheduler/context_test.go

```diff
@@ -255,6 +255,49 @@ func TestContext_AddNode(t *testing.T) {
 	}
 }
 
+func TestContext_AddRMBuildInformation(t *testing.T) {
+	context := createTestContext(t, pName)
+
+	rmID1 := "myCluster1"
+	BuildInfoMap1 := make(map[string]string)
+	BuildInfoMap1["buildDate"] = "2006-01-02T15:04:05-0700"
+	BuildInfoMap1["buildVersion"] = "latest"
+	BuildInfoMap1["isPluginVersion"] = "false"
+	rm1 := &si.RegisterResourceManagerRequest{
+		RmID:        rmID1,
+		PolicyGroup: "policygroup",
+		Version:     "0.0.2",
+		BuildInfo:   BuildInfoMap1,
+	}
```

Comment:
Leverage the `SetRMInfos()` call that you have added. Create the test context as you did. Call `SetRMInfos()` with the buildInfoMap created here.
The first one should give a length of 1 etc (asserts like you have)
Create a second buildInfoMap object call `SetRMInfos()` again make sure that you have two objects in the list.
Update a value in one of the two maps (like buildVersion is 1.0.0) Call `SetRMInfos()` again with the updated map and make sure it is replaced as expected.
That will give us a full test of the code